Wikipedia/Categories/UIImage+WMFStyle.m

```diff
@@ -14,4 +14,15 @@ + (UIImage*)wmf_placeholderImage {
     return img;
 }
 
++ (instancetype)wmf_imageFromColor:(UIColor*)color {
+    CGRect rect = CGRectMake(0, 0, 1, 1);
```

Comment:
if we don't want to do a shared cache object, we could at least wrap something domain-specific in a dispatch once (much like the other color categories). e.g. `+ (UIImage*)wmf_languageSelectionBackgroundImage`